platform_versioned_docs/version-23.4/cli/commands.mdx

```diff
@@ -374,15 +383,19 @@ Run `tw compute-envs -h` to view the list of supported compute environment opera
   The CLI can directly launch pipelines that have not been added to the Launchpad in a Platform workspace by using the full pipeline repository URL:
 
   ```console
-  tw launch https://github.com/nf-core/rnaseq --params-file=./custom_rnaseq_params.yaml \
-  --compute-env=my_aws_ce --revision 3.8.1 --profile=test,docker  
+  tw launch https://github.com/nf-core/rnaseq \
+  --params-file=./custom_rnaseq_params.yaml \
+  --config=<path/to/nextflow/conf/file> \
+  --compute-env=my_aws_ce --revision 3.8.1 \
+  --profile=test,docker  
 
     Workflow 2XDXxX0vCX8xhx submitted at user workspace.
 
-      https://tower.nf/user/user1/watch/2XDXxX0vCX8xhx
+      https://cloud.seqera.io/user/user1/watch/2XDXxX0vCX8xhx
   ```
 
   - Pipeline parameters are defined within the `custom_rnaseq_params.yaml` file.
+  - The optional `--config` flag is used to pass a custom Nextflow configuration file — configuration values passed here override the same values in the default pipeline `nextflow.conf`.
```

Comment:
In this case the comment is precise because there is no Platform pipeline object in the middlle.